nextcloudappstore/core/static/assets/js/account-api-token.js

```diff
@@ -0,0 +1,96 @@
+(function (global) {
+    'use strict';
+
+    function regenAuthToken(url, api_token) {
+        let request = new Request(
+            url,
+            {
+                method: 'POST',
+                headers: new Headers({
+                    'Content-Type': 'application/json',
+                    'Authorization': 'Token ' + api_token
+                }),
+            }
+        );
+        return fetch(request).then(global.convertResponse);
+    };
+
+
+    function showElement(element, boolean) {
+        if (boolean) {
+            element.removeAttribute('hidden');
+        } else {
+            element.setAttribute('hidden', 'true');
+        }
+    }
+
+
+    function showTokenFetchFailureMessage(boolean) {
```

Comment:
Thats also the reason why you use enumerations instead of simple integers :) Poka yoke!